packages/liquidator/test/Liquidator.js

```diff
@@ -964,6 +973,157 @@ contract("Liquidator.js", function(accounts) {
+          assert.deepStrictEqual((await emp.getLiquidations(sponsor1))[0].state, LiquidationStatesEnum.UNINITIALIZED);
+
+          // The other two positions should not have any liquidations associated with them.
+          assert.deepStrictEqual(await emp.getLiquidations(sponsor2), []);
+        });
+        it("should trigger and only submit the emp minimum position", async () => {
+          const liquidatorConfig = {
+            // entire fund dedicated to strategy
+            whaleDefenseFundWei: toWei("90"),
+            // will extend even if withdraw progress is 0% (typically this would be set to 50% +)
+            defenseActivationPercent: 0
+          };
+          const liquidator = new Liquidator({
+            logger: spyLogger,
+            expiringMultiPartyClient: empClient,
+            gasEstimator,
+            votingContract: mockOracle.contract,
+            syntheticToken: syntheticToken.contract,
+            priceFeed: priceFeedMock,
+            account: accounts[0],
+            empProps,
+            config: liquidatorConfig
+          });
+          // sponsor1 creates a position with 120 units of collateral, creating 100 synthetic tokens.
+          await emp.create({ rawValue: convert("120") }, { rawValue: toWei("100") }, { from: sponsor1 });
+
+          // liquidatorBot creates a position to have synthetic tokens to pay off debt upon liquidation.
+          // does not have enough to liquidate entire position
+          await emp.create({ rawValue: convert("1000") }, { rawValue: toWei("90") }, { from: liquidatorBot });
+
+          // Start with a mocked price of 1 usd per token.
+          // This puts both sponsors over collateralized so no liquidations should occur.
+          priceFeedMock.setCurrentPrice("1");
+          // sponsor withdraws below collat ratio
+          await emp.requestWithdrawal({ rawValue: convert("10") }, { from: sponsor1 });
+          await liquidator.update();
+          await liquidator.liquidatePositions();
+
+          // There should be exactly one liquidation in sponsor1's account. The liquidated collateral should be the original
+          // amount of collateral minus the collateral withdrawn. 120 - 10 = 110
+          let liquidationObject = (await emp.getLiquidations(sponsor1))[0];
+          assert.equal(liquidationObject.sponsor, sponsor1);
+          assert.equal(liquidationObject.liquidator, liquidatorBot);
+          assert.equal(liquidationObject.state, LiquidationStatesEnum.PRE_DISPUTE);
+          // minimum liquidation size / outstanding tokens * (outstanding collateral - withdraw amount)
+          // 5 / 100 * ( 120 -10) = 5.5
+          assert.equal(liquidationObject.liquidatedCollateral, convert("5.5"));
+          // minimum liquidation size / outstanding tokens * outstading collateral
+          // 5 / 100 * 120 = 6
+          assert.equal(liquidationObject.lockedCollateral, convert("6"));
+        });
+      });
     });
   }
 });
```

Comment:
I cant comment on the file below but what's the point of `packages/liquidator/test/strategy.js`?

Reply:
left in by mistake